cli/integration_tests/run_logging/log_prefix.t

```diff
@@ -0,0 +1,60 @@
+Setup
+  $ . ${TESTDIR}/../setup.sh
+  $ . ${TESTDIR}/setup.sh $(pwd) ./monorepo
+
+# Run for the first time with --log-prefix=none
+  $ ${TURBO} run build --log-prefix=none
```

Comment:
Add a test for `${TURBO} run build --log-prefix=junk`

Reply:
eagle eye chris! I thought of this too and figured i'd add a test if someone called it out. I was thinking error on junk/missing value was too harsh.